bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -305,7 +305,7 @@ impl EventTimelineItem {
     }
 
     pub fn raw(&self) -> Option<String> {
```

Comment:
Shouldn't we rename this one as well?

Reply:
It is removed in the second commit, so I didn't bother renaming it first.